aspnetcore/tutorials/your-first-mac-aspnet.md

```diff
@@ -54,6 +54,10 @@ Your project is now created, you can use the following commands to get going
        dotnet run
    ```
 
+> [!WARNING]
+> .NET Core has transitioned to *.csproj* project files in 1.0-preview3 and later. At this time, generator-aspnet does not generate a *.csproj* project file nor include the *global.json* required to specify the desired SDK version for compatibility. If you are deploying to an Azure App Service or have a more recent .NET Core SDK than 1.0-preview2 installed, see the generator-aspnet [Version Info](https://github.com/OmniSharp/generator-aspnet/wiki/Version-info) and [Issue #836](https://github.com/OmniSharp/generator-aspnet/issues/836) for more information.
```

Comment:
How about something like this?:

The ASP.NET Core generator currently generates projects that require 1.0.0-preview2 of the .NET Core SDK to build and are not compatible with more recent versions of the .NET Core SDK. To ensure your project builds with the correct version of the .NET Core SDK specify the correct [SDK version](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json#sdk) to use in your *global.json* file. See the generator-aspnet [Version Info](https://github.com/OmniSharp/generator-aspnet/wiki/Version-info) and [Issue #836](https://github.com/OmniSharp/generator-aspnet/issues/836) for more information.